tests/Propel/Tests/Resources/blog-database.xml

```diff
@@ -35,23 +35,23 @@
       <parameter name="unique_constraint" value="true"/>
     </behavior>
   </table>
-  <table name="blog_author" phpName="BlogAuthor" package="Acme.Blog" namespace="Acme\Model\Blog" baseClass="Acme\Model\ActiveRecord" description="The list of authors">
+  <table name="blog_author" phpName="BlogAuthor" package="Acme.Blog" namespace="\Acme\Model\Blog" baseClass="Acme\Model\ActiveRecord" description="The list of authors">
     <column name="id" phpName="Id" type="smallint" size="3" primaryKey="true" autoIncrement="true" required="true"/>
     <column name="username" phpName="Username" type="varchar" size="15" required="true"/>
     <column name="password" phpName="Password" type="varchar" size="40" required="true"/>
     <unique name="author_password_unique_idx">
       <unique-column name="username" size="8"/>
     </unique>
   </table>
-  <table name="blog_category" phpName="BlogCategory" package="Acme.Blog" namespace="Acme\Model\Blog" baseClass="Acme\Model\ActiveRecord" description="The list of categories">
+  <table name="blog_category" phpName="BlogCategory" package="Acme.Blog" namespace="\Acme\Model\Blog" baseClass="Acme\Model\ActiveRecord" description="The list of categories">
```

Comment:
Do we need to adjust the presention here?
Couldnt we adjust the Interpretation instead to be fqcn?

Reply:
I don't understand what you mean, could you please elaborate?

If you mean to always interpret the `namespace` attribute as fully qualified, no matter the trailing backslash, that would remove the relative namespace feature, which I think is quite nice.

Reply:
But what about the baseLcass config?

    baseClass="Acme\Model\ActiveRecord"

vs 

    baseClass="\Acme\Model\ActiveRecord"
    
Would this not be inconsistent to keep that one then without leading slash?

Reply:
Yes, that makes sense. Consistency is king. I changed it, so `baseClass` and `baseQueryClass` always start with a backslash.
When generating model files, the preceding slash will be removed (in `AbstractOmBuilder::declareClass()`, tested in `AbstractOmBuilderNamespaceTest::testGetDeclareClass()`, same as before.
Users can still use either format.

Reply:
Are there any BC breaks to expect from changing the syntax on the XML files?
Would existing projects and declarations run into some troubles? Or is all compatible?

Reply:
The changes should not affect existing schema.xml files or model classes generated from it.

The only actual change is when Propel writes a schema.xml file, which I think is only ever done during `database:reverse`. Then you get a different, more "understandable" output now (with all the absolute vs relative namespaces issue). And no more doubly namespaces. The resulting model classes do not differ.